templates/Makefile

```diff
@@ -40,6 +40,7 @@ teardown-remote-state:
 teardown-secrets:
```

Comment:
What do you think about prompting the user if they are sure before running any of the teardown steps?

Reply:
yeah perhaps a good idea since the terraform steps warn them before doing so 🤔